packages/keybr-keyboard/lib/layout.ts

```diff
@@ -890,9 +890,54 @@ export class Layout implements XEnumItem {
       Geometry.MATRIX,
     ),
   );
+  static readonly TH_TH = new Layout(
+    /* id= */ "th-th",
+    /* xid= */ 0x9a,
+    /* name= */ "Thai Kedmanee",
+    /* family= */ "qwerty",
+    /* language= */ Language.TH,
+    /* emulate= */ true,
+    /* geometries= */ new Enum(
+      Geometry.ANSI_101,
+      Geometry.ANSI_101_FULL,
+      Geometry.ISO_102,
+      Geometry.ISO_102_FULL,
+      Geometry.MATRIX,
+    ),
+  );
+  static readonly TH_PAT = new Layout(
+    /* id= */ "th-pat",
+    /* xid= */ 0x9b,
+    /* name= */ "Thai Pattachote",
+    /* family= */ "qwerty",
+    /* language= */ Language.TH,
+    /* emulate= */ true,
+    /* geometries= */ new Enum(
+      Geometry.ANSI_101,
+      Geometry.ANSI_101_FULL,
+      Geometry.ISO_102,
+      Geometry.ISO_102_FULL,
+      Geometry.MATRIX,
+    ),
+  );
+  static readonly TH_MAN = new Layout(
+    /* id= */ "th-man",
+    /* xid= */ 0x9c,
+    /* name= */ "Thai Manoonchai V1",
+    /* family= */ "qwerty",
+    /* language= */ Language.TH,
+    /* emulate= */ true,
+    /* geometries= */ new Enum(
+      Geometry.ANSI_101,
+      Geometry.ANSI_101_FULL,
+      Geometry.ISO_102,
+      Geometry.ISO_102_FULL,
+      Geometry.MATRIX,
+    ),
+  );
   static readonly RO_RO = new Layout(
     /* id= */ "ro-ro",
-    /* xid= */ 0xa0,
```

Comment:
Please keep the old ID! I make a mistake by skipping a few IDs before the Romanian layout. But these IDs must be stable, should never change.